projects/frontend/data-pipelines/gui/e2e/integration/lib/explore/data-job-details.spec.js

```diff
@@ -0,0 +1,158 @@
+
+describe('Data Job Explore Details Page', { tags: ['@dataPipelines', '@exploreDataJobDetails'] }, () => {
+    let dataJobExploreDetailsPage;
+    let testJobs;
+
+    before(() => {
+        return DataJobExploreDetailsPage.recordHarIfSupported()
+                                        .then(() => cy.clearLocalStorageSnapshot('data-job-explore-details'))
+                                        .then(() => DataJobExploreDetailsPage.login())
+                                        .then(() => cy.saveLocalStorage('data-job-explore-details'))
+                                        .then(() => cy.cleanTestJobs())
+                                        .then(() => cy.prepareBaseTestJobs())
+                                        .then(() => cy.fixture('lib/explore/test-jobs.json'))
+                                        .then((loadedTestJobs) => {
+                                            testJobs = applyGlobalEnvSettings(loadedTestJobs);
+
+                                            return cy.wrap({ context: 'explore::data-job-details.spec::before()', action: 'continue' });
+                                        });
+    });
+
+    after(() => {
+        cy.cleanTestJobs();
+
+        DataJobExploreDetailsPage.saveHarIfSupported();
+    });
+
+    beforeEach(() => {
+        cy.restoreLocalStorage('data-job-explore-details');
+
+        DataJobExploreDetailsPage.initBackendRequestInterceptor();
+    });
+
+    it('Data Job Explore Details Page - should load and show job details', () => {
+        cy.log('Fixture for name: ' + testJobs[0].job_name);
+
+        const dataJobsExplorePage = DataJobsExplorePage.navigateTo();
+
+        dataJobsExplorePage
+            .openJobDetails(testJobs[0].team, testJobs[0].job_name);
+
+        dataJobExploreDetailsPage = DataJobExploreDetailsPage
+            .getPage();
+
+        dataJobExploreDetailsPage
+            .getDetailsTab()
+            .should('be.visible');
+
+        dataJobExploreDetailsPage
+            .getMainTitle()
+            .should('be.visible')
+            .should('contains.text', testJobs[0].job_name);
+
+        dataJobExploreDetailsPage
+            .getStatusField()
+            .should('be.visible')
+            // TODO: do the right assertion (once it gets implemented)
+            .should('have.text', 'Not Deployed');
+
+        dataJobExploreDetailsPage
+            .getDescriptionField()
+            .should('be.visible')
+            .should('contain.text', testJobs[0].description);
+
+        dataJobExploreDetailsPage
+            .getTeamField()
+            .should('be.visible')
+            .should('have.text', testJobs[0].team);
+
+        dataJobExploreDetailsPage
+            .getScheduleField()
+            .should('be.visible')
+            .should('contains.text', 'At 12:00 AM, on day 01 of the month, and on Friday');
+
+        // DISABLED Because there is no Source at the moment.
```

Comment:
To be honest, these TODOs have been there for ages, and apparently we did not find important to discuss/resolve them. 

I would suggest to remove them from the code and discuss once we find the need to do so.